### DISCUSSION TITLE
Play back voice messages faster and in background

### DISCUSSION THREAD

**Discussion by Duvio** [element-hq/element-meta#1922] [Ideas]
In some messengers like WhatsApp, you can play back voice messages at 1.5 and 2 times the actual speed if wanted. This is an option that no messenger should miss, as listening to long voice messages can be tedious.

Additionally, one should be able to leave the app and have the voice message still playing in background.

**Comment by Yoneo** [comment]
Great idea! Additionally it would be great, if the voice message continues playing when the phone is rotated (vertical to horizontal or horizontal to vertical).
